database/dtb.ksy

```diff
@@ -0,0 +1,128 @@
+meta:
+  id: dtb
+  title: Flattened Devicetree Format
+  file-extension: dtb
+  application:
+    - Linux
+    - Das U-Boot
+  xref:
+    - wikidata: Q16960371
+  license: CC0-1.0
+  endian: be
+doc: |
+  Also referred to as Devicetree Blob (DTB). It is a flat
+  binary encoding of data (primarily devicetree data, although
+  other data is possible as well).
+
+  On Linux systems that support this the blobs can be accessed in
+  /sys/firmware/fdt
+
+  https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-firmware-ofw
+
+  The encoding of strings used in the strings block and struct block is
+  actually a subset of ASCII:
+
+  https://github.com/devicetree-org/devicetree-specification/blob/master/source/chapter2-devicetree-basics.rst
+
+  Example files:
+
+  - https://github.com/qemu/qemu/tree/master/pc-bios
+doc-ref:
+  - https://github.com/devicetree-org/devicetree-specification/releases/tag/v0.3
+  - https://github.com/devicetree-org/devicetree-specification/blob/ba2aa679679fc4fedf67130f18a6f0ecc4cf0382/source/flattened-format.rst
+  - https://elinux.org/images/f/f4/Elc2013_Fernandes.pdf
+seq:
+  - id: magic
+    contents: [0xd0, 0x0d, 0xfe, 0xed]
+    -orig-id: magic
+  - id: total_size
+    type: u4
+    -orig-id: totalsize
+  - id: structure_block_offset
+    type: u4
+    -orig-id: off_dt_struct
+  - id: strings_block_offset
+    type: u4
+    -orig-id: off_dt_strings
+  - id: memory_reservation_block_offset
+    type: u4
+    -orig-id: off_mem_rsvmap
+  - id: version
+    type: u4
+    -orig-id: version
+  - id: last_compatible_version
+    type: u4
+    valid:
+      max: version
+    -orig-id: last_comp_version
+  - id: boot_cpuid_phys
+    type: u4
+    -orig-id: boot_cpuid_phys
+  - id: strings_block_size
+    type: u4
+    -orig-id: size_dt_strings
+  - id: structure_block_size
+    type: u4
+    -orig-id: size_dt_struct
+instances:
+  memory_reservation_block:
+    pos: memory_reservation_block_offset
+    size: structure_block_offset - memory_reservation_block_offset
+  structure_block:
+    pos: structure_block_offset
+    size: strings_block_offset - structure_block_offset
+    type: fdt_block
+  strings_block:
+    pos: strings_block_offset
+    size: strings_block_size
+    type: strings
+types:
+  strings:
+    seq:
+      - id: strings
+        type: strz
+        encoding: ASCII
```

Comment:
Given that all strings in the spec use `ASCII` encoding, what about making it default in `/meta/encoding` instead of repeating it all around the file?